pkg/aws/tags/tags.go

```diff
@@ -46,3 +46,6 @@ const Environment = prefix + "environment"
 
 // AdminRole tags the role as admin (true/false)
 const AdminRole = prefix + "admin_role"
+
+// RedHatManaged tags the role as red_hat_managed
+const RedHatManaged = prefix + "red_hat_managed"
```

Comment:
This should be `red-hat-managed` (hyphens instead of underscores)

Reply:
Also, remove the `prefix`, since we want `red-hat-managed` to be the entire tag.

Reply:
@vkareh Made the changes you requested, please feel free to resolve this thread if it looks good!